scripts/install/lazylibrarian.sh

```diff
@@ -0,0 +1,162 @@
+# https://lazylibrarian.gitlab.io/install/
+# https://www.reddit.com/r/LazyLibrarian/comments/jw5e22/lazylibrarian_calibre_calibreweb_booksonic/
+# https://github.com/lazylibrarian/LazyLibrarian
+# https://gitlab.com/LazyLibrarian/LazyLibrarian
+# https://lazylibrarian.gitlab.io/config_commandline/
+# https://github.com/swizzin/swizzin/pull/743
+# https://lazylibrarian.gitlab.io/config_users/
+# https://lazylibrarian.gitlab.io/config_downloaders/
+# https://www.reddit.com/r/LazyLibrarian/
+# https://lazylibrarian.gitlab.io/api/
+
+##########################################################################
+# Import Sources
+##########################################################################
+
+. /etc/swizzin/sources/functions/color_echo
+. /etc/swizzin/sources/functions/pyenv
+. /etc/swizzin/sources/functions/users
+. /etc/swizzin/sources/functions/utils
+
+##########################################################################
+# Variables
+##########################################################################
+
+app_name="lazylibrarian"
+pretty_name="LazyLibrarian"
+default_port="5299"
+master=$(_get_master_username)
+data_dir="/home/$master/.config/lazylibrarian"
+app_dir="/opt/$app_name"
+venv_dir="/opt/.venv/$app_name"
+
+##########################################################################
+# Functions
+##########################################################################
+
+function _dependencies() {
+    echo_info "Installing dependencies for $pretty_name..."
+
+    echo_progress_start "Creating $pretty_name venv"
+    mkdir -p "$venv_dir"
+    python3_venv "$master" "$app_name"
+    echo_progress_done
+
+    echo_progress_start "Installing python dependencies to venv"
+    # shellcheck disable=2154           # log variable is inherited from box itself.
+    "$venv_dir/bin/pip" install --upgrade pip >>"${log}" 2>&1 # Upgrade PIP
+    PIP='apprise pyOpenSSL'
+    # shellcheck disable=2086           # We want the $PIP variable to expand. So 2086's warning is invalid here.
+    "$venv_dir/bin/pip" install $PIP >>"${log}" 2>&1
+    chown -R "$master": "$venv_dir"
+    echo_progress_done
+    # TODO: https://lazylibrarian.gitlab.io/config_rtorrent/ says we would require libtorrent for rtorrent compatability
+}
+
+function _install() {
+    echo_progress_start "Installing $pretty_name..."
+    mkdir -p "$app_dir"
+    mkdir -p "$data_dir"
+    #    Git clone/extract LL wherever you like
+    # shellcheck disable=2154           # log variable is inherited from box itself.
+    git clone "https://gitlab.com/LazyLibrarian/LazyLibrarian.git" "$app_dir" >>"$log" 2>&1
+    if [[ ! -e "$app_dir/LazyLibrarian.py" ]]; then
+        ecoh_error "Git clone unsuccessful. Try running the box install again."
+        exit 1
+    fi
+    chown -R "$master": "$app_dir" # Change owner\group recursively for new dirs.
+    chown -R "$master": "$data_dir"
```

Comment:
Need to add a non-recusrive chown on `~/.config` so it won't be owned by root

Reply:
If #746 is merged first, maybe no need

Reply:
Added it in anyways. Couldn't hurt.